src/textual/_asyncio.py

```diff
@@ -16,7 +16,7 @@
 else:
     import asyncio
     from asyncio import create_task as _create_task
-    from typing import Awaitable
+    from typing_extensions import Awaitable
```

Comment:
Why the change, here? The docs show that `typing.Awaitable` exists since Python 3.5 and it evens says (if I'm reading it right) that it has been deprecated in favour of `collections.abc.Awaitable`. ([source](https://docs.python.org/3/library/typing.html#typing.Awaitable))

Reply:
The result of some automated search and replacing.